src/main/java/org/monarchinitiative/biodownload/DownloadableResource.java

```diff
@@ -0,0 +1,81 @@
+package org.monarchinitiative.biodownload;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+/**
+ * Class used to store a downloadable resource and with all the resources objects that can be downloaded.
+ */
+public class DownloadableResource {
```

Comment:
We can add `equals()`, `hashCode`, and `toString` methods here.

Reply:
resolved in 5c6e06